google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -52,12 +52,26 @@
 import java.util.concurrent.TimeUnit;
 
 /**
- * A Google cloud storage object.
+ * A Google cloud storage object. A {@code Blob} object includes the {@code BlobId} instance, the
```

Comment:
Be very careful with the word "object". This is very easy to confuse wit an instance of the Storage class. I suggest using the phrase "an object in Google Cloud Storage" when you mean the bytes on the server rather than "storage object" which sounds like an instance of a class.

The overloading of the word "object" is really annoying. GCS should never have called these things "objects" but that ship, I am afraid, has sailed and we can't fix it here. :-(

Reply:
agree